packages/shared/src/pathToRegexp.ts

```diff
@@ -1,12 +1,36 @@
-import { pathToRegexp as pathToRegexpBase } from './compiled/path-to-regexp';
+import type {
+  Match,
+  MatchFunction,
+  ParseOptions,
+  Path,
+  RegexpToFunctionOptions,
+  TokensToRegexpOptions,
+} from './compiled/path-to-regexp';
+import { match as matchBase, pathToRegexp as pathToRegexpBase } from './compiled/path-to-regexp';
 
 export const pathToRegexp = (path: string) => {
   try {
     // @ts-ignore no types exists for the pre-compiled package
     return pathToRegexpBase(path) as RegExp;
```

Comment:
Interesting: the pre-commit hooks (likely `npx eslint --fix`) removes the `as RegExp`. I had to skip the pre-commit hooks to avoid that diff. Is this necessary?

Reply:
Ah I see now - the type declaration is in `index.d.ts` now, so we don't need it here! Fixed.

061c5557bfd8f65efb08b2c5554194fabb841091